src/EditorFeatures/VisualBasicTest/BasicEditorServicesTest.vbproj

```diff
@@ -13,6 +12,7 @@
     <TargetFramework>net461</TargetFramework>
     <RuntimeIdentifier>$(RoslynDesktopRuntimeIdentifier)</RuntimeIdentifier>
     <RoslynProjectType>UnitTest</RoslynProjectType>
+    <RootNamespace></RootNamespace>
```

Comment:
Intentional?

Reply:
This is sadly necessary now. The SDK forces a default namespace on the condition `'$(Namespace)' == ''`. Hence we can't disable it globally anymore because there is no flag to say "yes I deliberately don't want a namespace". Have to go to individual projects and disable it there.